frontend/packages/kubevirt-plugin/integration-tests/tests/vmi.list-filtering.scenario.ts

```diff
@@ -0,0 +1,63 @@
+import { browser } from 'protractor';
+import { appHost, testName } from '@console/internal-integration-tests/protractor.conf';
+import {
+  addLeakableResource,
+  createResource,
+  removeLeakedResources,
+} from '@console/shared/src/test-utils/utils';
+import { isLoaded } from '@console/internal-integration-tests/views/crud.view';
+import { getVMManifest } from './utils/mocks';
+import { VM_STATUS } from './utils/consts';
+import { filterBoxCount } from '../views/vms.list.view';
+import { VirtualMachine } from './models/virtualMachine';
+
+const waitForVM = async (
+  manifest: any,
+  status: VM_STATUS,
+  resourcesSet: Set<string>,
+  kind?: 'virtualmachines' | 'virtualmachineinstances',
+) => {
+  const vm = new VirtualMachine(manifest.metadata, kind || 'virtualmachines');
+
+  createResource(manifest);
+  addLeakableResource(resourcesSet, manifest);
+  await vm.waitForStatus(status);
+};
+
+const waitForVMList = async () => {
+  await browser.get(`${appHost}/k8s/ns/${testName}/virtualmachines`);
+  await isLoaded();
+};
+
+describe('Test List View Filtering (VMI)', () => {
+  const leakedResources = new Set<string>();
+  const testVM = getVMManifest('Container', testName, `${testName}-vm-test`);
+  const testVMI = getVMManifest(
+    'Container',
+    testName,
+    `${testName}-vmi-test`,
+    null,
+    `VirtualMachineInstance`,
+  );
+
+  beforeAll(async () => {
+    await waitForVM(testVM, VM_STATUS.Off, leakedResources);
```

Comment:
should this be really in before all block? Shouldn't we also check the vmi status change?

Reply:
> should this be really in before all block?

The test is to filter by status, knowing the the vmi is `Running` while the vm is `Off`
So we need one vm in `Off` status and one vmi in `Running` status, before we start the tests.

> Shouldn't we also check the vmi status change?

vmi is always running ( if evrything is ok :-) )

Reply:
ok then